saskatoon/member/models.py

```diff
@@ -73,6 +73,9 @@ class AuthUser(AbstractBaseUser, PermissionsMixin):
 
     is_staff = models.BooleanField(default=False, null=False)
 
+    # Default set to False for backwards compatibility
```

Comment:
Please remove comment, I find it confusing: default is set to False because new users don't have a temporary password (e.g. a volunteer or owner being added) not only backwards compatibility